plugins/highlight-keywords/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+<meta charset="utf-8" />
+<link rel="shortcut icon" href="favicon.png" />
+<title>Show Language ▲ Prism plugins</title>
+<base href="../.." />
+<link rel="stylesheet" href="style.css" />
+<link rel="stylesheet" href="themes/prism.css" data-noprefix />
+<script src="prefixfree.min.js"></script>
+<style>
+	/*
+	 * Custom keyword styles
+	 */
+	.token.keyword.keyword-return, .token.keyword.keyword-if{
+		color: #f92672;
+	}
+</style>
+<script>var _gaq = [['_setAccount', 'UA-33746269-1'], ['_trackPageview']];</script>
+<script src="http://www.google-analytics.com/ga.js" async></script>
+</head>
+<body>
+
+<header>
+	<div class="intro" data-src="templates/header-plugins.html" data-type="text/html"></div>
+
+	<h2>Highlight Keywords</h2>
+	<p>This plugins adds special CSS classes for each keyword matched in the code. For example, the keyword <code>if</code> will have the class <code>keyword-if</code>. You can have fine grained control over the appearance of each keyword by providing your own stylesheet.</p>
```

Comment:
"plugin", singular please :)

Also, "will have the class `keyword-if` **as well**". It will still have the class `keyword` too, right?

stylesheet -> CSS rules (stylesheet implies separate file, which is not necessary)

Apart from that, looks ok!